src/Form.js

```diff
@@ -23,6 +24,11 @@ class Form {
         for (const field in data) {
             this[field] = data[field];
         }
+
+        this.__options = { 
```

Comment:
The prefix __ is probably to mark it as private, but don't like how it looks.

Remove it 😀